src/main/webapp/dev/js/common/bootboxWrapper.js

```diff
@@ -15,29 +15,29 @@ const DEFAULT_CANCEL_TEXT = 'Cancel';
 const DEFAULT_YES_TEXT = 'Yes';
 const DEFAULT_NO_TEXT = 'No';
 
-function applyStyleToModal(modal, statusType) {
-    modal.find('.modal-header').addClass(`alert-${statusType || StatusType.DEFAULT}`)
-            .find('.modal-title').addClass(`icon-${statusType || StatusType.DEFAULT}`);
+function applyStyleToModal(modal, bootstrapContextualColors) {
```

Comment:
`bootstrapContextualColors` -> `bootstrapContextualColor`

Reply:
Right, my bad. Will do it straightaway.